config/purposes/ultima_pcr_free.wip.yml

```diff
@@ -10,15 +10,17 @@ UPF Shear:
   :asset_type: plate
 UPF Post Shear:
   :asset_type: plate
-UPF End Prep XP1:
+UPF End Prep:
```

Comment:
Might need to think through how this data change will work in UAT, where the purposes are already created.

I expect we'll end up with the 'UPF End Prep XP1' hanging around in UAT (as would be a pain to try to delete, linked to many objects), and this will create a new 'UPF End Prep' one on deployment.